docs_src/microservices/application/services/AppRecordReplay/GettingStarted.md

```diff
@@ -0,0 +1,210 @@
+    {
+        "running": false,
+        "eventCount": 20,
+        "duration": 20015604050,
+        "repeatCount": 2,
+        "Message": ""
+    }
+    
+    Replay canceled response::
+    {
+        "running": false,
+        "eventCount": 2,
+        "duration": 0,
+        "repeatCount": 0,
+        "Message": "replay canceled"
+    }
+    
+    No Replay response:
+    {
+        "running": false,
+        "eventCount": 0,
+        "duration": 0,
+        "repeatCount": 0,
+        "Message": "no replay running or previously run"
+    }
+    ```
+
+#### Cancel Replay
+
+A replay session can be canceled while it is running. Review the **Cancel Replay** DELETE API in the [API Reference](ApiReference.md) section and use the **Cancel Replay** request from the PostMan collection referenced above. Press `Send` in PostMan to cancel the replay session.
+
+!!! note
+    This API will return a **202 - Accepted** response if the replay can be canceled, otherwise it will return an error such as "***failed to cancel replay: no replay currently running***"
+
+### Export a Recorded Session
+
+The current recorded session can be exported so that data can be saved to the file system. Review the **Export** GET API in the [API Reference](ApiReference.md) section and use the **Export Recording** requests from the PostMan collection referenced above.
+
+This API exports all the events, related devices and device profiles. It has an optional `compression` query parameter. Valid values are `none`, `gzip` and `zlib` . Defaults to `none` if not specified. 
+
+!!! note
+    Use the Postman `Send and Download` option for the `Export recording - JSON` request so that the response can be saved to file. The `Send and Download` option is on the `Send` button.
+
+!!! note
+    Postman automatically un-compresses the responses when requesting GZIB or ZLIB compression. Use the following curl commands to save the compressed response to file.
+    
+    ```text
+    curl localhost:59712/api/{{api_version}}/data?compression=gzip -o recording.gz
+    curl localhost:59712/api/{{api_version}}/data?compression=zlib -o recording.zlib
+    ```
+
+### Import a Record Session
+
+This API allows a previously exported recording to be imported back into the service. Review the **Import** POST API in the [API Reference](ApiReference.md) section and use the **Import Recording** requests from the PostMan collection referenced above.
+
+This API has a the optional `overwrite` query parameter, which specifies to overwrite existing Devices and Device Profiles or not. Defaults to true if not set.
+
+!!! note
+    Only one recording is save in memory at a time. Importing will overwrite the current recoding if one exists.
+
+!!! warning
+    The source device service(s) must be running will importing and the `overwite` parameter above is `true`. This is because the device service(s) are sent messages when the devices and profiles in the imported data are added to the system.
```

Comment:
```suggestion
    The source device service(s) must be running when importing and the `overwrite` parameter above is `true`. This is because the device service(s) are sent messages when the devices and profiles in the imported data are added to the system.
```

Reply:
fixed